src/utils/notification-template.ts

```diff
@@ -50,11 +52,12 @@ export const render = (transport: TransportName, notification: Notification, var
     formatAnswer,
     getRealityQuestionLink,
     getSnapshotProposalLink,
+    unsubscribeEmail,
   };
 
   return ejs.renderFile(path, context, {
     cache: true,
-    async: true,
+    async: false,
```

Comment:
This caused me some headache. It theorically should not be required, but somehow if I let the async happen, the rendered result contains `[Object promise]` in every place an EJS `include` happens. The code that is calling this function has an await, so I have no clue of what is causing this behavior. Forcing synchronous rendering fixed it.